doc/source/getting-started.rst

```diff
@@ -0,0 +1,194 @@
+.. _getting-started:
+
+*********************
+Getting Started Guide
+*********************
+
+The following guide will step through an example of developing and testing a
+new Ansible role. After reading this guide, you should be familiar with the
+basics of how to use Molecule and what it can offer.
+
+.. note::
+
+    In order to complete this guide by hand, you will need to additionally
+    install `Docker`_.
+
+.. _Docker: https://docs.docker.com/
+
+Creating a new role
+-------------------
+
+Molecule uses `Ansible Galaxy`_ under the hood to generate conventional role
+layouts. If you've ever worked with roles before, you'll be right at home. If
+not, please review the `Role directory structure`_ guide to see what each
+folder is responsible for.
+
+.. _Ansible Galaxy: https://docs.ansible.com/ansible/latest/reference_appendices/galaxy.html#the-command-line-tool
+.. _Role directory structure: https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#role-directory-structure
```

Comment:
I'm curious whether it's possible to reference this using intersphinx. I think it's more flexible this way.

Reply:
Hooked up InterSphinx! Much better, thanks for recommending.